include/bouncer.h

```diff
@@ -54,6 +54,8 @@ enum SocketState {
 	CL_LOGIN,		/* login_client_list */
 	CL_WAITING,		/* pool->waiting_client_list */
 	CL_WAITING_LOGIN,	/*   - but return to CL_LOGIN instead of CL_ACTIVE */
+	CL_WAITING_LOGICAL_REP,	/* pool->waiting_logical_replication_list */
+	CL_WAITING_PHYSICAL_REP,	/* pool->waiting_physical_replication_list */
```

Comment:
Why do we need separate lists for logical and physical replication?

Reply:
I guess you're right and we probably don't need that (after changing a few things).